assets/src/edit-story/app/index.js

```diff
@@ -40,4 +41,5 @@ export {
   useMedia,
   useSnackbar,
   useCanvas,
+  useUser,
```

Comment:
Not sure about the naming here.

`useCurrentUser` and `CurrentUserProvider` seem more descriptive.

Thoughts?

Reply:
I kept it generic to user, as we could move the user logic from the author dropdown here, keep all user data together

Reply:
Hmm I see. Not sure if that's a great fit 🤔 

@miina What would be your thought on that?

Reply:
Not sure either. Is the idea moving everything related to the authors and users here then? Currently, the logic is in the `inspectorProvider` since it's only really relevant for that area. I suppose it could all be in the `userProvider` for the sake of grouping everything related to the users together under one name, this would move the author-related stuff much higher in the hierarchy without any reason though. Or is the author's logic needed outside of the inspector as well?

Reply:
It also different from other providers that are named after the type of object.  Like media. Why is this different?

Reply:
Not sure I understand the question. Are you basically asking why the name of the `inspectorProvider` is not `authorProvider`? Since it includes other relevant things for the inspector as well and not just the author-related things. (If you look at lower hierarchy providers outside of the `app` folder there are quite a few that are not directly related to an object)

Reply:
@miina @swissspidy Neither of your comments showed up when I put my last response. 

My point was we could move a function like [loadUsers](https://github.com/google/web-stories-wp/blob/413b113b1f81eba2dcc0d3988d10e20a5889f955/assets/src/edit-story/components/inspector/inspectorProvider.js#L112-L127) to this new user provide **IF** it makes sense. If you don't want to do this, then fine, just a way to keep users all together. 

I don't care to much about this, so I will change it to currentUser. I originally had it this way, it just looked weird, as it is two words other providers are one. It looked weird when I put it in. But willing to change it.

Reply:
Sounds a lot like premature optimization :-) I don't think we need it at this point.

And if we really want it eventually, renaming is super cheap.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6132/commits/1512e0a4e6d4e9260c7b72e4a693d0a4309a0897